pkg/storage/storage_test.go

```diff
@@ -96,6 +96,24 @@ func Test_svc_GetLastListedObj(t *testing.T) {
 			}
 		}
 	}
+	r.NoError(storage.CleanLastListedObj(ctx, s1.FromStorage, s1.ToStorage, b1))
+	for i, stor := range stors {
+		for j, buck := range bucks {
+			for k, pref := range prefs {
```

Comment:
Would be easier to read if indices would be named as `storeIdx`, `jucketIdx`, `prefIdx`

Reply:
fixed